MaterialDesignThemes.UITests/WPF/PasswordBoxes/PasswordBoxTests.cs

```diff
@@ -83,42 +84,36 @@ public async Task OnPasswordBox_WithClearButton_ClearsPassword()
         }
 
         [Fact]
-        [Description("PR 2828")]
+        [Description("PR 2828 and Issue 2930")]
```

Comment:
The original test did not catch the issues, because it updated the VM as the first thing. Doing that kick-started the bindings and then changes in the UI works afterwards.

So I have changed the test a bit, and I now have 2 tests: one for the non-reveal style and another for the reveal style.